OrderService/OrderService.API/Controllers/PaymentController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OrderService.Application.UseCases.PaymentUseCases.PayOrderUseCase;
+
+namespace OrderService.API.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class PaymentController : ControllerBase
+{
+    private readonly IMediator _mediator;
+
+    public PaymentController(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+
+    [HttpGet]
+    [Route("pay")]
+    [Authorize]
+    public async Task<ActionResult<string>> Pay(string orderId)
```

Comment:
specify where you getting this id from using attributes. E.g. fromQuerry, fromBody e.t.c.